### DISCUSSION TITLE
PostgreSQL LIMIT statement definition

### DISCUSSION THREAD

**Discussion by enesgur** [grafana/grafana#42892] [Data Sources]
Hi there,

I'm using PostgreSQL as a data source. Sometimes, the user queries didn't include the "LIMIT" statement. That's why the Grafana CPU was up to over %90.

I don't know, maybe before the query, we can use the `explain` command for the query. 

Do you have any plan for that?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
